main_routers/characters_router.py

```
@@ -1126,34 +1490,83 @@ async def rename_catgirl(old_name: str, request: Request):
+            initialize_character_data = get_initialize_character_data()
+            await initialize_character_data()
+
+            memory_server_reloaded = await notify_memory_server_reload(
+                reason=f"角色重命名: {old_name} -> {new_name}",
+            )
+        except MaintenanceModeError:
+            await _rollback_character_operation(
+                _config_manager,
+                characters_snapshot=characters_snapshot,
+                memory_snapshot_records=memory_snapshot_records,
+                reason=f"维护模式：角色重命名回滚 {old_name} -> {new_name}",
```

Ruff (0.15.9)

[warning] String contains ambiguous `：` (FULLWIDTH COLON). Did you mean `:` (COLON)?

(RUF001)